src/SFA.DAS.EmployerAccounts/Configuration/ConfigurationKeys.cs

```diff
@@ -12,5 +12,6 @@ public static class ConfigurationKeys
         public const string EmployerFinance = "SFA.DAS.EmployerFinance";
         public const string EmployerAccountsReadStore = "SFA.DAS.EmployerAccounts.ReadStore";
         public const string AuditApi = "SFA.DAS.AuditApiClient";
+        public static string ReservationsClientApiConfiguration => $"SFA.DAS.Reservations.Api.Client";
```

Comment:
should all these Configuration Keys be consistent ? either as const or static properties

Reply:
Yes , good point.  Changed